components/scanners/snyk-sbom/scanner/Dockerfile

```diff
@@ -0,0 +1,154 @@
+# Start with a base image
```

Comment:
Is there a ready image for this? It looks a little overkill to having to bake this